### DISCUSSION TITLE
Minimise to tray

### DISCUSSION THREAD

**Discussion by ProFire** [cdgriffith/FastFlix#430] [Ideas]
May I request for minimise to tray ability? When I share my screen in a remote meeting, I don't want to show everyone that I'm running Fastflix.

**Comment by cdgriffith** [comment]
I was avoiding this because of how hard I was afraid it would be after adding those windows notifications was a pain. 

However, seems like PySide6 made it super easy, barely an inconvenience! Works well on Ubuntu, will test on Windows and see how it interacts with the notification system already in place, and hopefully in next release!

**Comment by cdgriffith** [comment]
Added in https://github.com/cdgriffith/FastFlix/releases/tag/5.4.0 thanks for the idea!
